spec/support/content_data_api.rb

```diff
@@ -119,12 +119,6 @@ def content_data_api_has_document_types
         stub_request(:get, url).to_return(status: 200, body: body)
       end
 
-      def content_data_api_has_csv
```

Comment:
We should be explicit when setting up the tests. It is better not to hide the response with the method, so it can be reused in different places:

I would suggest something like:

```ruby
  def content_data_api_has_csv(columns: [], values: [])
  end
```

And then use it with:

```ruby
content_data_api_has_csv(
  columns: %w(Date Value), 
  values: [['10-10-2018',1000],[['11-10-2018',1100]])
```
To be consistent with what we already have, I would put this method in `content_data_api`

Reply:
Good idea! 👍